tools/update_repo.sh

```diff
@@ -30,7 +30,7 @@ function get_os_dists {
     elif [ "$os" == "el" ]; then
         alldists='6 7 8'
     elif [ "$os" == "fedora" ]; then
-        alldists='27 28 29 30 31 32 33'
+        alldists='27 28 29 30 31 32 33 34'
```

Comment:
@Totktonada Should I delete previous versions here?

Reply:
I think it can be useful if we'll need to manually push something to an old repository. So would keep old distros.

Reply:
Nit: I would move the update_repo commit before adding the workflow: this way each commit would be fully working.

Reply:
update_repo commit was moved